docs/rules/object-shorthand.md

```diff
@@ -0,0 +1,46 @@
+# Require Object Literal Shorthand Syntax (object-shorthand)
+
+EcmaScript6 provides a more concise form for defining object literal methods and properties
+including the following forms:
+
+```js
+var x = {
+    go() {},
+    *x() {}
```

Comment:
Missing a comma.

Now that we have loaders, should we start linting example code in the docs?